doltpy/core/dolt.py

```diff
@@ -88,14 +87,14 @@ def config(self, is_global, user_name, user_email):
             _execute(args=name_args, cwd=self.repo_dir)
             _execute(args=email_args, cwd=self.repo_dir)
 
-    def clone(self, repo):
-        # if self.dir_exists:
-        #     raise Exception(self.repo_dir + " .")
-        #
-        # os.makedirs(self.repo_dir)
-        # self.dir_exists = True
+    def clone(self, repo_url: str):
+        if self.dir_exists:
+            raise Exception(self.repo_dir + " .")
```

Comment:
Would be good to get a better exception message here.

Reply:
Good shout, thanks Brian

Reply:
```(ld-analytics) [oscarbatori:analytics(oscarbatori/analytics)] >> sync-events-dolthub --window-end 2020/03/01/00 --hours-lookback 4 --source s3 --s3-bucket liquidata-events-prod --remote-url Liquidata/product-analytics --branch separate-tables
Traceback (most recent call last):
  File "/Users/oscarbatori/anaconda3/envs/ld-analytics/bin/sync-events-dolthub", line 10, in <module>
    sys.exit(events_dolthub_sync())
  File "/Users/oscarbatori/anaconda3/envs/ld-analytics/lib/python3.7/site-packages/ld_analytics/events_api/sync.py", line 93, in main
    remote_url='Liquidata/product-analytics')
  File "/Users/oscarbatori/anaconda3/envs/ld-analytics/lib/python3.7/site-packages/doltpy/etl/dolthub_loader.py", line 26, in loader
    repo.clone(remote_url)
  File "/Users/oscarbatori/anaconda3/envs/ld-analytics/lib/python3.7/site-packages/doltpy/core/dolt.py", line 93, in clone
    raise Exception(self.repo_dir + " .")
Exception: /var/folders/86/pvj3p5x12j1bzm740hrs45r00000gp/T/tmpzyqry0yl .```

Prescient you were... from just a few minutes ago.